internal/models/handler/user.go

```diff
@@ -1,6 +1,4 @@
-package model
-
-import "github.com/porky256/mock-interview-tbr/internal/skill/model"
+package handlermodel
```

Comment:
I think it's better to keep the model package and just rename structures like UserDB and UserResponse

Reply:
Agree, but why response? We may use it in requests too. What about UserHandler?

Reply:
You're right

Reply:
done, resolve it if you are satisfied